src/app_service/service/network/service.nim

```diff
@@ -36,40 +35,32 @@ proc newService*(events: EventEmitter, settingsService: settings_service.Service
   result.settingsService = settingsService
 
 proc fetchNetworks*(self: Service) =
-  let response = backend.getEthereumChains()
+  let response = backend.getFlatEthereumChains()
   if not response.error.isNil:
-    raise newException(Exception, "Error getting combinedNetworks: " & response.error.message)
-  let combinedNetworksDto = if response.result.isNil or response.result.kind == JNull: @[]
-            else: Json.decode($response.result, seq[CombinedNetworkDto], allowUnknownFields = true)
-  self.combinedNetworks = combinedNetworksDtoToCombinedItem(combinedNetworksDto)
-  self.flatNetworks = @[]
-  for network in self.combinedNetworks:
-    self.flatNetworks.add(network.test)
-    self.flatNetworks.add(network.prod)
+    raise newException(Exception, "Error getting networks: " & response.error.message)
```

Comment:
@dlipicar because of this, we should have a related `catch` block, otherwise, the app will crash.

Reply:
Hmm interesting case. Changed the exception with an error at the moment, though not being able to get the list of networks from the backend is kind of a critical failure that should never happen (there's only a DB access involved, no networks, so something must've gone super wrong if we get an error there and the app will most likely misbehave).